plugins/amazon_eks.yaml

```diff
@@ -190,7 +190,7 @@ pipeline:
         info: i
         warning: w
```

Comment:
the opentelemetry alias for `warning` is `warn`, I think `warning` will make the parser fail to build. The best part is, it'll fail to build it silently, and just return a "default" parser :)